samples/idtokens-serverless.js

```diff
@@ -12,32 +12,34 @@
 // limitations under the License.
 
 // sample-metadata:
-//   title: ID Tokens for Cloud Run
-//   description: Requests a Cloud Run URL with an ID Token.
-//   usage: node idtokens-cloudrun.js <url> [<target-audience>]
+//   title: ID Tokens for Serverless
+//   description: Requests a Cloud Run or Cloud Functions URL with an ID Token.
+//   usage: node idtokens-serverless.js <url> [<target-audience>]
 
 'use strict';
 
 function main(
   url = 'https://service-1234-uc.a.run.app',
   targetAudience = null
 ) {
-  // [START google_auth_idtoken_cloudrun]
+  // [START google_auth_idtoken_serverless]
+  // [START run_service_to_service_auth]
+  // [START functions_bearer_token]
   /**
    * TODO(developer): Uncomment these variables before running the sample.
    */
-  // const url = 'https://YOUR_CLOUD_RUN_URL.run.app';
+  // const url = 'https://TARGET_URL';
   const {GoogleAuth} = require('google-auth-library');
   const auth = new GoogleAuth();
 
   async function request() {
     if (!targetAudience) {
-      // Use the request URL hostname as the target audience for Cloud Run requests
+      // Use the request URL hostname as the target audience for requests.
       const {URL} = require('url');
       targetAudience = new URL(url).origin;
     }
     console.info(
-      `request Cloud Run ${url} with target audience ${targetAudience}`
```

Comment:
Looks like you need to run `npm run fix` and re-push - should help get the linter happy :)